lib/plugins/features/movies/controller.js

```diff
@@ -8,3 +8,27 @@ exports.create = (payload) => {
     return new Movie({ id: movie.id }).fetch();
   });
 };
+
+exports.list = (queryOptions) => {
+  let sortProperty;
+  let sortDirection;
+  const query = queryOptions.q;
+
+  if (queryOptions.sort.charAt(0) === '-') {
+    sortProperty = queryOptions.sort.slice(1);
+    sortDirection = 'DESC';
+  } else {
+    sortProperty = queryOptions.sort;
+    sortDirection = 'ASC';
+  }
+
+  return new Movie()
+  .query((qb) => {
+    qb.whereRaw('lower(title) LIKE ?', `%${query}%`);
```

Comment:
optional: you could use `qb.whereRaw('lower(title) LIKE %?%', query);` to keep all the psql operators on the left side and remove the need for string interpolation.